nion/swift/DisplayPanel.py

```diff
@@ -1868,9 +1872,17 @@ def close() -> None:
                 command = workspace_controller.remove_display_panel(self)
                 document_controller.push_undo_command(command)
 
+        def edit() -> None:
+            action_context = document_controller._get_action_context()
+            action_context.display_panel = self
+            action_context.display_item = self.display_item
+            document_controller.perform_action_in_context("window.open_title_edit", action_context)
+
+
```

Comment:
Should only be a single blank line here. Please consider using `flake8 --ignore=E501 nion/swift/Panel.py nion/swift/DisplayPanel.py` and addressing any issues specific to this PR. You may need to install `conda install flake8` in your environment.